app/assets/javascripts/angular/controllers/desktop/terror.js

```diff
@@ -0,0 +1,62 @@
+(function() {
```

Comment:
As above, this is broken off from the main file now. This is the terror tracker for the main desktop dashboard.